diff --git a/node_modules/whatwg-url/lib/url-state-machine.js b/node_modules/whatwg-url/lib/url-state-machine.js
index c25dbc2..8c2d955 100644
--- a/node_modules/whatwg-url/lib/url-state-machine.js
+++ b/node_modules/whatwg-url/lib/url-state-machine.js
@@ -1,5 +1,5 @@
 "use strict";
-const punycode = require("punycode");
+const punycode = require("punycode/");
 const tr46 = require("tr46");
 
 const specialSchemes = {
